src/transformers/convert_slow_tokenizer.py

```diff
@@ -893,12 +893,42 @@ def converted(self) -> Tokenizer:
         return tokenizer
 
 
+class BlenderbotConverter(Converter):
+    def converted(self) -> Tokenizer:
+        ot = self.original_tokenizer
+        vocab = ot.encoder
+        merges = list(ot.bpe_ranks.keys())
+
+        tokenizer = Tokenizer(
+            BPE(
+                vocab=vocab,
+                merges=merges,
+                dropout=None,
+                continuing_subword_prefix="",
+                end_of_word_suffix="",
+                fuse_unk=False,
+            )
+        )
+
+        tokenizer.pre_tokenizer = pre_tokenizers.ByteLevel(add_prefix_space=ot.add_prefix_space)
+        tokenizer.decoder = decoders.ByteLevel()
+        tokenizer.post_processor = processors.TemplateProcessing(
+            single=f"$A:0 {ot.sep_token}:0",
+            special_tokens=[
+                (ot.sep_token, ot.sep_token_id),
```

Comment:
As the slow version uses the end of sentence token (`eos_token`) in the `build_inputs_with_special_tokens` method, shouldn't we use that one here too?
```suggestion
                (ot.eos_token, ot.eos_token_id),
```

Reply:
Good catch! Fixed it :)